plugins/ssr/ssrast.mli

```diff
@@ -140,6 +140,9 @@ type 'tac ssrhint = bool * 'tac option list
 type 'tac fwdbinders =
         bool * (ssrhpats * ((ssrfwdfmt * ast_closure_term) * 'tac ssrhint))
 
+type 'tac ffwbinders =
+  (ssrhpats * ((ssrfwdfmt * ast_closure_term) * 'tac ssrhint))
```

Comment:
Doesn't this factor with the above `fwbinders` as `type 'a fwdbinders = bool * 'tac ffwbinders`?